policies/communication_channels.md

```diff
@@ -0,0 +1,62 @@
+# Communication channels in the Astropy Project
+
+## Preface
+
+Communication in the Astropy Project is governed by the [Community
+Code of Conduct](https://www.astropy.org/code_of_conduct.html), which
+takes precedence to anything written here. This goal of this policy is
+to write down some of the technicalities of how email lists, slack
+channels, forums etc. are set up.
+
+While Astropy aims to be an inclusive and welcoming community, we
+recognize that not all users want to be involved in the technical
+details of building and running a library like astropy, thus there are
+communication channels that are more geared towards users and others
+to developers. Also, in a project this size, certain issues that
+arise require privacy (e.g. personal or financial information is
+legally protected in many contries). Thus, there are communication
+channels geared towards or exclusively for developers, maintainers, or
+members of certain committees.
+
+
+## email communication geared towards project members
+
+The following mailing lists are maintained as Google groups. Unless
+specifically restricted, anyone worldwide can subscribe to the
+following lists and previous posts are publically avilable. List
+membership is required to post messages to prevent spam.
```

Comment:
```suggestion
The following mailing lists are maintained as Google groups. Unless
specifically restricted, anyone worldwide can subscribe to these lists,
and previous posts are publicly available. List
membership is required to post messages to prevent spam.
```